proto/src/main/proto/Rss.proto

```diff
@@ -334,4 +337,16 @@ message AccessClusterRequest {
 message AccessClusterResponse {
   StatusCode status = 1;
   string retMsg = 2;
+  StorageConfItem storageConf = 3;
+}
+
+message FetchClientConfResponse {
+  StatusCode status = 1;
+  string retMsg = 2;
+  StorageConfItem storageConf = 3;
```

Comment:
FetchClientConf should be common api, it's not for storage conf only.

Reply:
Yes, `StorageConfItem` is just a wrapper of storage related conf and we only have storage conf for now. We may add other conf filed, say ComputeConfItem in the furture.

Reply:
It should be a common feature at first step, and there shouldn't have concept like StorageConfItem, ComputeConfItem, etc. ConfigurationItem should include all of them.

Reply:
As discussed offline, we will maintain a separate conf file contains kv pairs (eg, spark.rss.storage.type LOCALFILE) in this version @colinmjj @jerqi @hzlzh